autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillManagementRecyclerAdapter.kt

```diff
@@ -25,41 +25,66 @@ import androidx.recyclerview.widget.RecyclerView.Adapter
 import com.duckduckgo.app.browser.favicon.FaviconManager
 import com.duckduckgo.autofill.domain.app.LoginCredentials
 import com.duckduckgo.autofill.impl.databinding.ItemRowAutofillCredentialsManagementScreenBinding
+import com.duckduckgo.autofill.impl.databinding.ItemRowAutofillCredentialsManagementScreenHeaderBinding
 import kotlinx.coroutines.launch
 
 class AutofillManagementRecyclerAdapter(
     val lifecycleOwner: LifecycleOwner,
     val faviconManager: FaviconManager,
+    val grouper: CredentialGrouper,
     val onCredentialSelected: (credentials: LoginCredentials) -> Unit,
     val onCopyUsername: (credentials: LoginCredentials) -> Unit,
     val onCopyPassword: (credentials: LoginCredentials) -> Unit
-) : Adapter<AutofillManagementRecyclerAdapter.CredentialsViewHolder>() {
+) : Adapter<RecyclerView.ViewHolder>() {
 
-    private var credentials = listOf<LoginCredentials>()
+    private var listItems = listOf<ListItem>()
 
     override fun onCreateViewHolder(
         parent: ViewGroup,
         viewType: Int
-    ): CredentialsViewHolder {
-        val binding = ItemRowAutofillCredentialsManagementScreenBinding.inflate(LayoutInflater.from(parent.context), parent, false)
-        return CredentialsViewHolder(binding)
+    ): RecyclerView.ViewHolder {
+        return when (viewType) {
+            ITEM_VIEW_TYPE_HEADER -> {
+                val binding = ItemRowAutofillCredentialsManagementScreenHeaderBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+                HeadingViewHolder(binding)
+            }
+            ITEM_VIEW_TYPE_CREDENTIAL -> {
+                val binding = ItemRowAutofillCredentialsManagementScreenBinding.inflate(LayoutInflater.from(parent.context), parent, false)
```

Comment:
I was looking into `ItemRowAutofillCredentialsManagementScreenBinding` and the contents can be replaced by TwoLineListItem

Reply:
Will look at this in https://github.com/duckduckgo/Android/pull/2108